### ISSUE TITLE
[Bug]: [bulkload]num_entities is not correct if bulk load 32 files in one request

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#17152]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20220518-b9568177
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2):
```

### Current Behavior

collection.num_entities is not correct if bulkload 32 files in one request

### Expected Behavior

num_entities is correct

### Steps To Reproduce

```markdown
1. prepare 32 data files as attached
2. bulkload the files in one request
3. collection.num_entities
```

### Milvus Log

data files:
[Archive 2.zip](https://github.com/milvus-io/milvus/files/8750814/Archive.2.zip)

**Comment by soothing-rain** [comment]
/unassign
/assign @yhmo 

@yanliang567 Also could you provide some test scripts or at least schema examples? Thanks :)

**Comment by yanliang567** [comment]
it reproduces once(1/8) on master-20220602-48cf63a2 with 2 data nodes deployed.
[issue.17152.log.tar.gz](https://github.com/milvus-io/milvus/files/8841807/issue.17152.log.tar.gz)

**Comment by yhmo** [comment]
Fixed by #17384

**Comment by yanliang567** [comment]
verified fix on master-20220608-6d078ebe
